highLevelInterface/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ endif()
 set(SOURCES
 	calcAmplitude.cc
 	pwaFit.cc
+	pwaNloptFit.cc
```

Comment:
This addition should be removed (it is responsible for the Travis error).

`pwaNloptFit.cc` should only be added if `USE_NLOPT` is `true`, and that is already taken care of just below.